packages/flutter/lib/src/rendering/object.dart

```diff
@@ -1103,6 +1104,45 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
   /// Override in subclasses with children and call the visitor for each child.
   void visitChildren(RenderObjectVisitor visitor) { }
 
+  /// Capture an image of the current state of the render object and its
+  /// children.
+  ///
+  /// The returned [ui.Image] has uncompressed raw bytes in the dimensions of
+  /// the render object, multiplied by the [window.devicePixelRatio].
+  ///
+  /// The property [isRepaintBoundary] must be true for [toImage] to work, and
+  /// the render object must not need layout, painting, or compositing.
+  ///
+  /// The [pixelRatio] describes the scale between the logical pixels and the
+  /// size of the output image. It is independent of the
+  /// [window.devicePixelRatio] for the device, so specifying 1.0 (the default)
+  /// will give you a 1:1 mapping between logical pixels and the output pixels
+  /// in the image. The output image is not anti-aliased, so you may wish to
+  /// use a larger device pixel ratio to obtain an image large enough to
+  /// down sample. If you wish to have an image the same size as the actual
+  /// device pixels, then specify [window.devicePixelRatio] as the [pixelRatio].
+  ///
+  /// See also:
+  ///
+  ///  * [Scene.toImage] for more information about the image returned.
```

Comment:
Does this have to be `[ui.Scene.toImage]`?

Reply:
search for `[ui.` or `[dart:ui.` and compare to actual output, I forget which one is the right one.